README.md

```diff
@@ -2,13 +2,15 @@
 
 this is a webpack project that the purpose of this project is to understand how use callbacks and promises,
 learn how to use proper ES6 syntax,use ES6 modules to write modular JavaScript and use webpack to bundle JavaScript.
+
+![MyImage](./src/images/screenshot.png)
+
 # Built With
 - HTML
 - CSS
 - JavaScripth
 - Webpack
```

Comment:
- [ ] Kindly add instructions below this one on how someone will be able to start the app. Take note of the following:
- Cloning the repo
- cd into the directory
- installing the npm packages (`npm install`)
- And finally starting the app. (that will be `npm start`) 

Please add these instructions above below the built-with technologies so one will know to access and start your app. 👍